lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -225,6 +233,36 @@ def self.get_github_release_tag_names(repo_name)
           UI.user_error!("Please make sure it doesn't have any unsaved changes and delete it to continue.")
         end
       end
+
+      private_class_method def self.build_changelog_sections(changelog_sections)
+        changelog_sections.reject { |k, v| v.empty? }.map do |section_name, prs|
+          next unless prs.size > 0
+
+          case section_name
+          when :breaking_changes
+            title = "## Breaking Changes"
+          when :fixes
+            title = "## Bugfixes"
+          when :new_features
+            title = "## New Features"
+          when :other
+            title = "## Other Changes"
+          end
+          "#{title}\n#{prs.join("\n")}"
+        end.join("\n")
+      end
+
+      private_class_method def self.get_section_depending_on_types_of_change(change_types)
+        if change_types.include?("breaking")
+          :breaking_changes
+        elsif change_types.include?("feat")
+          :new_features
+        elsif change_types.include?("fix")
+          :fixes
+        else
+          :other
```

Comment:
I guess for SDKs where we are using this but not forcing the labels, most changes will be in `other`. I think it's ok though since we do offer the chance for devs to modify the generated changelog

Reply:
yes, we will use labels everywhere at some point,  but until them we will probably have to edit the changelogs manually